lms/static/js/instructor_dashboard/proctoring.js

```diff
@@ -7,15 +7,9 @@ $(function() {
     $proctoringAccordionPane.accordion(
         {
             heightStyle: 'content',
-            activate: function(event, ui) {
```

Comment:
I removed this stuff because it seemed impossible to subscribe to an event which would be called both on tab click and accordion creation. Well, it's almost possible, but there's a timing issue between when the ProctoredExamDashboard (from edx-proctoring) is created and when this accordion is created. 

It's a bad UI, anyway, and should be revisited.

Reply:
This code creates a cookie to remember which accordion tab you last visited. If you load the page a second time, it selects that tab when the accordion is created, rather than defaulting to the first tab. The problem is that I only want to load the iframe when the tab is selected. If I listen to the `accordionactivate` event, this works fine, *except* in the case when the accordion opens to the review dashboard immediately. In that case, `accordionactivate` isn't called at all. `accordioncreate` is called instead, but unfortunately that event happens before the listener in ProctoredExamDashboardView can be created...

ProctoredExamDashboardView is in https://github.com/edx/edx-proctoring/pull/467

Reply:
Thank you for the explanation.

Reply:
I see. Thank you for the explanation.